src/dags/dag_update.py

```diff
@@ -26,18 +26,18 @@ def doit():
 
     fetch_dags_from_github = BashOperator(
         task_id="fetch_dags_from_github",
-        bash_command=f"wget {github_url} -O /tmp/repo.zip",
+        bash_command=f"wget {github_url} -O {SHARED_DIR}/repo.zip",
     )
 
     extract_zip = BashOperator(
-        task_id="extract_zip", bash_command="unzip -o /tmp/repo.zip -d /tmp"
+        task_id="extract_zip", bash_command="unzip -o {SHARED_DIR}/repo.zip -d {SHARED_DIR}"
```

Comment:
Here, is an example. No format string, hence shell variable expansion (I presume), hence the need for a `$`.

Reply:
Good one, thanks.